### PR TITLE
Persona redirectoin

### PR DESCRIPTION
**Pull request by JanikOskar**
@laniszew @lmazurowski @

### CODE REVIEW

File: src/components/ContactBook/ContactBook.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import React, { FC, useState } from 'react';
+import styled from 'styled-components';
+import { useQuery } from '@apollo/react-hooks';
+import { gqlEntity } from 'global/graphqls/schema';
+import { Spinner } from 'components/Spinner/Spinner';
+import { Overlay } from 'components/Overlay/Overlay';
+import { HamburgerMenu } from '../../global/Layouts/HamburgerMenu/HamburgerMenu';
```
Comment (laniszew) [inline src/components/ContactBook/ContactBook.tsx]: use absolute import

File: src/components/ContactBook/ContactBook.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+import { PersonaCard } from '../PersonaCard/PersonaCard';
+import { GET_PERSONAS, GetPersonaType } from '../../global/graphqls/Persona';
```
Comment (laniszew) [inline src/components/ContactBook/ContactBook.tsx]: use absolute import

File: src/components/MyPersona/MyPersona.tsx [hunk 3]
```diff
@@ -16,6 +16,7 @@ import { Overlay } from 'components/Overlay/Overlay';
 import { NavLink } from 'react-router-dom';
 import AddIcon from 'assets/AddIcon.svg';
 import ShareQrCode from 'assets/ShareQrCode.svg';
+import { APP_ROUTES } from '../../global/AppRouter/routes';
```
Comment (laniszew) [inline src/components/MyPersona/MyPersona.tsx]: use absolute import

File: src/components/PersonaCard/EditAndRemoveMenu.tsx [hunk 4]
```diff
@@ -77,15 +86,40 @@ export const EditRemoveMenu: FC<EditAndRemoveMenuType> = ({ uuid }) => {
     }
   };
 
+  const RemoveFunctionality = () => {
+    if (pathname.includes('personas')) {
+      return (
+        <MenuRemoveStyled key="1">
+          <EditAndRemoveBox
+            onClick={async () => {
+              await personaRemove();
```
Comment (laniszew) [inline src/components/PersonaCard/EditAndRemoveMenu.tsx]: is await needed here?

File: src/components/PersonaCard/EditAndRemoveMenu.tsx [hunk 5]
```diff
@@ -77,15 +86,40 @@ export const EditRemoveMenu: FC<EditAndRemoveMenuType> = ({ uuid }) => {
...
+            <img src={RemoveIcon} alt="Remove Icon" />
+          </EditAndRemoveBox>
+        </MenuRemoveStyled>
+      );
+    } else {
+      return (
+        <MenuRemoveStyled key="1">
+          <EditAndRemoveBox
+            onClick={async () => {
+              await spotRemove();
```
Comment (laniszew) [inline src/components/PersonaCard/EditAndRemoveMenu.tsx]: is await needed here?

File: src/pages/Login/Login.tsx [hunk 6]
```diff
@@ -18,7 +17,7 @@ import { Link } from 'react-router-dom';
 import { PageWrapper } from 'components/PageWrapper/PageWrapper';
 // TODO: Remove after real integration
 import { useUserContext } from 'global/UserContext/UserContext';
-import { signInWithGoogle } from '../../helpers/signInWithGoogle';
```
Comment (laniszew) [inline src/pages/Login/Login.tsx]: why this is removed?

File: src/pages/Login/Login.tsx [hunk 7]
```diff
@@ -169,7 +181,7 @@ export const Login: FunctionComponent = () => {
                 )}
               </StyledCard>
               <RegisterCaption>
-                Don’t have account? <Link to={APP_ROUTES.REGISTER}>Register Now</Link>
+                Don’t have account? <Link to="/register">Register Now</Link>
```
Comment (laniszew) [inline src/pages/Login/Login.tsx]: why this is changed?

File: src/components/SpotBook/SpotBook.tsx [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+`;
+
+const DateOfSave = styled.div`
+  margin-bottom: 8px;
+`;
+
+export const SpotBook: FC = () => {
+  const { loading, data } = useQuery<GetCardType>(GET_SPOT);
+  const [searchValue, setSearchValue] = useState('');
+  if (loading) {
```
Comment (lmazurowski) [inline src/components/SpotBook/SpotBook.tsx:28]: one line space :)

File: src/components/SpotPage/SpotPage.tsx [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+type Props = {
+  card: EntityType;
+  uuid: string;
+  isWithEdit?: boolean;
+};
+
+const CardStyled = styled(Card)`
+  margin-top: 0px;
+`;
+const NameStyled = styled.div`
```
Comment (lmazurowski) [inline src/components/SpotPage/SpotPage.tsx:18]: one line space :)

File: src/global/graphqls/schema.ts [hunk 10]
```diff
@@ -56,3 +56,28 @@ export type gqlUser = {
...
+  page: EntityPage;
+  personaUUIDs: string[];
+  qrCodeLink: string;
+  networkList: string[];
+  recommendList: AgregatedPersona[];
+  contactBook: AgregatedPersona[];
+  visibilityList: AgregatedPersona[];
+  spotBook: string[];
+};
+export type AgregatedSpot = {
```
Comment (lmazurowski) [inline src/global/graphqls/schema.ts:72]: one line space :)

**Pull request merged**